components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -260,7 +260,8 @@ export class ReportingComponent implements OnInit, OnDestroy {
     this.downloadOptsVisible = false;
 
     const reportQuery = this.reportQuery.getReportQuery();
-    const filename = `${reportQuery.endDate.format('YYYY-M-D')}.${format}`;
+    const endDate = reportQuery.endDate.clone().utc().endOf('day');
```

Comment:
Same question as @susanev : Why add the `.endOf('day')`  and only on this one of the three?